### DISCUSSION TITLE
Git Sync: allow to configure self signed CA certs

### DISCUSSION THREAD

**Discussion by nzamani** [Kong/insomnia#5139] [Ideas]
When syncing with our git server we you get the following error because we have a self signed certificate in our server's cert chain (see screenshot below):

**self signed certificate in certificate chain**

When using git from the CLI, this can be solved as follows:

1. Download the corresponding self signed CA cert (your root cert) 
1. Execute `git config --global http."https://mygit.corp/".sslCAInfo "<path to downloaded ca cert file>"`

This however has no effect on Insomnia.
Is there a chance to have feature allowing to configure self signed certificates that are then trusted by Insomnia?

-Nabi

![image](https://user-images.githubusercontent.com/884161/188647932-5dc42bae-96d3-4e07-bdbe-dcd41687d1b5.png)

**Comment by dschulten** [comment]
One can work around this by disabling the option **Validate certificates** in the preferences section **Request/Response**. Note however that this opens up Insomnia to accept responses without checking the counterpart's certificate, which is generally a bad idea.
